plugins/operators/__init__.py

```
@@ -0,0 +1,2406 @@
+            required=False,
+            label="Description",
+            description="A dictionary of information about the field",
+            view=types.CodeView(),
+        )
+
+        info = ctx.params.get("info", None)
+
+        if info is not None:
+            try:
+                json.loads(info)
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)